src/CubicEoS.jl

```diff
@@ -15,7 +18,6 @@ include("dbload.jl")
 include("basic_thermo.jl")
 include("chempotential.jl")
 include("vt_stability.jl")
-include("vt_flash.jl")
-include("newton.jl")
+include("vt_flash/vt_flash.jl")
```

Comment:
I'm merged all sources related to vt_flash into separate directory. Source `vt_flash/vt_flash.jl` includes necessary inner files.